mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -65,6 +67,77 @@ class Completions(
     (values, result)
   end completions
 
+  private def toCompletionValues(
+      completion: Completion
+  ): List[CompletionValue.Compiler] =
+    val shouldAddSnippet = path match
+      // if the parent is an apply then we already have ()
+      case (fun) :: (appl: GenericApply) :: _ if appl.fun == fun =>
+        false
+      case (_: Import) :: _ => false
+      case _ :: (_: Import) :: _ => false
+      case _ => true
+
+    completion.symbols.flatMap(
+      completionsWithSuffix(_, completion.label, shouldAddSnippet)
+    )
+  end toCompletionValues
+
+  inline private def undoBacktick(label: String): String =
+    label.stripPrefix("`").stripSuffix("`")
+
+  private def findApplySymbols(denot: Denotation): List[Symbol] =
```

Comment:
```suggestion
  private def allOverloadedSymbols(denot: Denotation): List[Symbol] =
```

or something?
Indeed, this method is used for collecting all overloaded `apply` methods, but this method itself is more generic.

Reply:
Changed and moved to MetalsEnrichments